docs/advanced-topics/metrics.rst

```diff
@@ -0,0 +1,154 @@
+Workflow Metrics
+================
+
+.. note:: Dockstore Workflow Metrics is currently in preview mode. Also note that the screenshots below were taken on our staging site.
+
+What are workflow metrics?
+------------------------------------
+Workflow metrics are metrics of workflow executions on a platform. Metrics include things like execution status and the resources used during the execution. 
+
+Users are able to execute workflows on various platforms using Dockstore's Launch With feature, shown on the right side in the screenshot below.
+
+.. figure:: /assets/images/docs/submit-metrics/workflow-launch-with.png
+    :alt: Launch With Partners
+
+Platforms are able to submit metrics of workflows executed on their platform to Dockstore and we aggregate the metrics and display them in the UI to the users in the :ref:`Metrics tab <How to view metrics>`.
+
+Why would I want to submit workflow metrics?
+--------------------------------------------
+As a platform owner, workflow metrics indicate to others that your platform is compatible with many workflows on Dockstore. Workflow metrics provide valuable information to users because it helps indicate that the workflow is high quality and is very likely to work for others.
+
+How do I submit workflow metrics?
```

Comment:
Suggest moving this down. I think the primary target for the page should be the end-user; there will hopefully be lots of users interested in this. Whereas the number of platforms submitting metrics, even if we get submissions from all of them, will be small.

As a more extreme suggestion, I'd even suggest moving the submission of metrics into a separate page.

On the other hand, this is a preview feature and we are trying to gather more metrics than present them at this point, so it could wait.